plotly/tests/test_optional/test_offline/test_offline.py

```diff
@@ -4,41 +4,21 @@
 """
 from __future__ import absolute_import
 
-import os
-from nose.plugins.attrib import attr
 from nose.tools import raises
 from unittest import TestCase
 
 import plotly
-from plotly.exceptions import PlotlyError
-
-dummy_js_url = ('https://gist.githubusercontent.com/chriddyp/'
-                'f40bd33d1eab6f0715dc/raw/'
-                '24cd2e4e62ceea79e6e790b3a2c94cda63510ede/'
-                'test.js')
```

Comment:
Just wondering if we want to keep these since we're only deprecating this functionality, not removing it yet?